.github/workflows/python-package.yml

```
@@ -25,7 +25,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, 'ci skip') || !contains(github.event.head_commit.message, 'skip ci')"
     strategy:
       matrix:
-        python-version: ['3.7', '3.7.5', '3.8']
+        python-version: ['3.7', '3.7.5', '3.8', '3.9']
```

```suggestion
        python-version: ['3.7', '3.8', '3.9']
```

As I suggested previously, remove the `py-3.7.5` build

---

Yes, this was part of #129